.github/workflows/cd.yml

```diff
@@ -0,0 +1,129 @@
+name: cd
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  get-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get.outputs.version }}
+    steps:
+      - id: get
+        run: |
+          $(echo "${{ github.event.release.tag_name }}" | sed 's/v//g') >> $GITHUB_OUTPUT
+
+  build-container:
+    runs-on: ubuntu-latest
+    needs:
+      - get-version
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-central-1
+
+      # Authenticate with ECR
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Login to GHCR
+        id: login-ghcr
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u USERNAME --password-stdin
```

Comment:
what is GHCR used for here?

Reply:
I'm going to publish the containers to GHCR so that the public can use them too